### ISSUE TITLE
Where is the Pcds_1000/0089_edt.pt?

### ISSUE THREAD

**Issue by lw0210** [FengQiaojun/TerrainMesh#1]
Dear ,
Hello, I run this training code, and the error shows that there is no Pcds_ 1000/0089_ Edt.pt, I checked Pcds_ 1000 and Pcds_ 1000_gt . There is no such folder in _ edt.pt file.And where is the split file?

**Comment by FengQiaojun** [comment]
Thanks for the interest on our project!
Sorry the edt files were not uploaded. We include a script on how to calculate edt from the sparse depth image [here](https://github.com/FengQiaojun/TerrainMesh/blob/semantic/mesh_init/run_calc_edt.py). By some modification you should be able to generate the edt.pt files needed.
Hope you find this useful!

**Comment by lw0210** [comment]
Thank you ！I will try it. @FengQiaojun

**Comment by lw0210** [comment]
Hi,  it also show val/013_68/Pcds_1000/0054_mesh1024.png ,how can I got the   0054_mesh1024.png. @FengQiaojun

**Comment by lw0210** [comment]
Thanks！I‘ll try it sonn. @FengQiaojun

**Comment by lw0210** [comment]
Dear，
I'm sorry to bother you again. I was delayed because of something.  When I  generate the  x_mesh1024.png，it  display  there is no Semantics_5/0085.png.Where is the Semantics_5 folder？And can I add your WeChat if you are convenient？Because I'm worried about other unknown mistakes. @FengQiaojun

**Comment by FengQiaojun** [comment]
Thank you for the question! I need to admit that the training code and data is not well organized.
The **Semantics_5** contains the groundtruth semantic labels for the 2D images. I did miss this in the original version of the uploaded dataset and I am going to update it this week!
Feel free to shoot me an email at qjfeng@ucsd.edu regarding the other issue like WeChat :)

**Comment by FengQiaojun** [comment]
Hi @lw0210 
Sorry for the delay but I have updated the dataset with the missing Semantics_5. Please check this [Google Drive link](https://drive.google.com/drive/folders/1AKFAIMIy66irr5cflrfKADXU_CZnYEKP?usp=sharing).
